pcsx2/ImGui/ImGuiManager.h

```diff
@@ -16,8 +17,16 @@ enum class InputLayout : u8;
 
 namespace ImGuiManager
 {
-	/// Sets the path to the font to use. Empty string means to use the default.
-	void SetFontPath(std::string path);
+	struct FontInfo
+	{
+		std::span<const u8> data;
+		std::span<const u32> exclude_ranges;
+		const char* face_name;
+		bool is_emoji_font;
+	};
+
+	/// Sets a list of fonts to use.
+	void SetFonts(std::vector<FontInfo> info);
 
```

Comment:
Given how we have to tread emoji fonts separately, would it make sense to pass those fonts as a second parameter instead of having to check `is_emoji_font` eveytime we iterate `info`?

Reply:
I mentioned the filtering as more of an example of translations working around ImGui issues, rather then anything to be address. (It looks fine to me).

Your patch does look reasonable, when I to a poke at it I had looked at `TryLoadFonts`/`DownloadFontIfMissing` instead https://github.com/TheLastRar/pcsx2/commit/122c5fb493f29ec5e5dd7e702ea572c844fbaa23

Reply:
Not a huge fan of either patch tbh, but I'd prefer mine if we have to use one

Reply:
I guess it's fine

Maybe a 3rd opinion would serve as a deciding factor.